deployments/deploy.sh

```diff
@@ -20,15 +20,6 @@ sudo usermod -a -G docker ubuntu
 
 docker pull znicoll/filter-it-image-processor
 
-sudo apt-get install jq -y
-
-TOKEN=`curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600"`
-META_DATA=`curl -H "X-aws-ec2-metadata-token: $TOKEN" -v http://169.254.169.254/latest/meta-data/iam/security-credentials/ec2_exec_role`
-
-export AWS_ACCESS_KEY_ID=$(jq ".AccessKeyId" <<< $META_DATA)
-export AWS_SECRET_ACCESS_KEY=$(jq ".SecretAccessKey" <<< $META_DATA)
-export AWS_SESSION_TOKEN=$(jq ".Token" <<< $META_DATA)
```

Comment:
These credentials were expiring and therefore breaking the containers - apparently it can just work based on the associated IAM role with the instance!